src/store/users/handlers.ts

```diff
@@ -0,0 +1,24 @@
+import * as api from './api';
+import usersActions from './slice';
+import { IUsersItem } from './types';
+
+type ISetUsersAction = () => any;
+
+export const setUsersAction: ISetUsersAction = () => (dispatch: any) => api.usersApi()
```

Comment:
all handlers should start from 'handle' and finish with 'action'
f.e, handleSetUsersAction
apply it everywhere